tests/root1/proc/vmstat

```diff
@@ -134,3 +134,7 @@ balloon_deflate 0
 balloon_migrate 0
 swap_ra 0
 swap_ra_hit 0
+pgpromote_success 168979
+pgdemote_kswapd 0
+pgdemote_direct 0
+pgdemote_khugepaged 0
```

Comment:
Make sure that all the 4 fields vary (and not only `pgpromote_success`) so that we get something else than `0.00` in the `demote/s` column of the report.